qupath-gui-fx/src/main/java/qupath/lib/gui/scripting/DefaultScriptEditor.java

```diff
@@ -526,7 +538,40 @@ public Stage getStage() {
 	public ObservableValue<Boolean> scriptRunning() {
 		return runningTask.isNotNull();
 	}
-	
+
+	/**
+	 * Update the font size for the script editor and console.
+	 */
+	protected void promptToSetFontSize() {
+		String current = fontSize.getValue();
+		if (current != null && current.length() - current.replaceAll(";", "").length() == 1) {
+			// Usually we store the style in a standard way, but sometimes we store a style exactly
+			current = current.replaceAll("-fx-font-size: ", "")
+					.replaceAll(";", "")
+					.strip();
+		}
+		String output = Dialogs.showInputDialog("Set font size",
+				"Enter the font size, in points or as a percentage \n(e.g. 16, 120%)", current);
+		if (output == null) {
+			return;
+		}
+		String style = null;
+		output = output.strip();
+		if (!output.isBlank()) {
+			if (output.contains("-fx"))
+				style = output;
+			else
+				style = "-fx-font-size: " + output;
+			if (!style.endsWith(";"))
+				style = style + ";";
+		}
+		// This only works if no other style is set!
+		fontSize.setValue(style);
+		paneCode.setStyle(style);
+		paneConsole.setStyle(style);
+	}
```

Comment:
Shouldn't you check that `style != null` before those 3 lines?

Reply:
I'd actually like style to be `null` - that should reset the style so that it depends upon any existing css/stylesheets. So the default behavior should be a style of null.